.ostree/packages-runtime-CentOS-10.txt

```diff
@@ -0,0 +1,3 @@
+kmod-kvdo
+python3-blivet
+vdo
```

Comment:
Shouldn't this be also commented out for now?

Reply:
commenting stuff out of these files is not supported, so I just removed them